XmlFormatter/src/Settings/Adapter/PropertyAdapter.cs

```diff
@@ -0,0 +1,164 @@
+            try
+            {
+                var propertyValue = Properties.Settings.Default[name];
+                returnValue = new SettingPair(name);
+                returnValue.SetValue(propertyValue);
+            }
+            catch (Exception)
+            {
+            }
+            
+            return returnValue;
+        }
+
+        /// <summary>
+        /// Get all the settings stored in the resource file
+        /// </summary>
+        /// <returns>All the settings present in the resource file</returns>
+        public List<ISettingPair> GetSettings()
+        {
+            List<ISettingPair> settingPairs = new List<ISettingPair>();
+            foreach (SettingsProperty property in Properties.Settings.Default.Properties)
+            {
+                if (property.Attributes.Contains(typeof(ApplicationScopedSettingAttribute)))
+                {
+                    continue;
+                }
+                SettingPair settingPair = new SettingPair(property.Name);
+                settingPair.SetValue(Properties.Settings.Default[property.Name]);
+                settingPairs.Add(settingPair);
+            }
+            return settingPairs;
+        }
+
+        /// <summary>
+        /// This method is not working for this adapter
+        /// </summary>
+        /// <param name="scope">The scope to add</param>
+        public void AddSubScope(ISettingScope scope)
+        {
+        }
+
+        /// <summary>
+        /// This method is not working for this adapter
+        /// </summary>
+        /// <returns>A new empty list</returns>
+        public List<ISettingScope> GetSubScopes()
+        {
+            return new List<ISettingScope>();
+        }
+
+        /// <summary>
+        /// This method is not working for this adapter
+        /// </summary>
+        /// <param name="name">The name of the sub scope to get</param>
+        /// <returns>null</returns>
+        public ISettingScope GetSubScope(string name)
+        {
+            return null;
+        }
+
+        /// <summary>
+        /// This method is not supported by this adapter
+        /// </summary>
+        /// <param name="name">The name of the scope to remove</param>
+        /// <returns>Always true</returns>
+        public bool RemoveSubScope(string name)
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// Method is not supported by the adapter
+        /// </summary>
+        public void ClearSubScopes()
+        {
+        }
+
+        /// <summary>
+        /// This method is not supported by this adapter
+        /// </summary>
+        /// <param name="name">The name of the setting to remove</param>
+        /// <returns>Always true</returns>
+        public bool RemoveSetting(string name)
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// Method is not supported by the adapter
+        /// </summary>
+        public void RemoveSettings()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053335)